Tests/SWBLLBuildTests/LLBuildTests.swift

```diff
@@ -616,7 +612,8 @@ fileprivate class LoggingDelegate: BuildSystemDelegate {
         // Configure the delegate
         class Delegate: LoggingDelegate {
             override func shouldCommandStart(_ command: Command) -> Bool {
-                assert(super.shouldCommandStart(command))
```

Comment:
The entire `assert`, including the call to the superclass implementation, would get optimized out, so the test would always fail in release mode - moved it to a separate statement to prevent this, and switched to `#expect` as well.